src/studio/src/designer/frontend/packages/ux-editor/src/App.tsx

```diff
@@ -22,11 +24,31 @@ import { textResourcesPath } from 'app-shared/api-paths';
 export function App() {
   const dispatch = useDispatch();
   const { org, app } = useParams();
+  const languageCode = 'nb';
+  const [searchParams, setSearchParams] = useSearchParams();
+  const layoutOrder = useSelector(
+    (state: IAppState) => state.formDesigner.layout.layoutSettings.pages.order
+  );
+  const selectedLayout = useSelector(
+    (state: IAppState) => state.formDesigner.layout.selectedLayout
+  );
+
+  // Set Layout to first layout in the page set if none is selected.
+  useEffect(() => {
+    if (!searchParams.has('layout') && layoutOrder[0]) {
+      setSearchParams({ ...deepCopy(searchParams), layout: layoutOrder[0] });
```

Comment:
Burde ikke være nødvendig med `deepCopy` her heller. Og `'layout'` kan kanskje plasseres i en variabel? ;-)
```suggestion
      setSearchParams({ ...searchParams, layout: layoutOrder[0] });
```
Ser det er flere steder hvor den samme endringen kan gjøres.

Reply:
Det fungerer ikke searchParams er en instrance av en klasse og kan ikke spreades.

Reply:
men den dype kopien kan spreades?

Reply:
yupp